examples/nodejs/aws/eventbridge/deploy-and-run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
if you're going to name the other file `and-run-cli.sh` then this one should probably be `deploy-and-run-web-app.sh`.